lib/courses/data/content.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/foundation.dart';
+import 'package:json_annotation/json_annotation.dart';
+import 'package:schulcloud/core/data.dart';
+
+part 'content.g.dart';
+
+enum ContentType {
+  text,
+  etherpad,
+  nexboad,
+  unknown,
+}
+
+@JsonSerializable()
+class Content extends Entity<Content> {
+  final Id<Content> id;
+  final String title;
+  final String text;
+  final String url;
+  final ContentType type;
+
+  Content(
+      {@required this.id,
+      @required this.title,
+      this.text,
+      this.url,
+      @required this.type})
```

Comment:
By adding a trailing comma, the formatting gets much nicer, especially the curly brace at the start